Expensify/App
# [No QA] Add NewDot deployer responsibilities

### Details
Add a couple new checkboxes to the StagingDeployCash to add some extra NewDot deployer responsibilites.

### Fixed Issues
$ https://github.com/Expensify/Expensify/issues/203091 https://github.com/Expensify/Expensify/issues/203088

### Tests
1. CP this pull request (if deploy checklist is locked). Otherwise just merge it without CP label.
1. The StagingDeployCash should get the new checklist items.
1. Check off the new items.
1. CP another pull request.
1. The items should be unchecked.
1. Check off the items again.
1. Mark an issue as a deploy blocker.
1. The items should remain checked.
